HelloApi/HelloApi/App_Start/WebApiConfig.cs

```diff
@@ -11,6 +13,12 @@ public static void Register(HttpConfiguration config)
         {
             // Web API configuration and services
 
+            var statsdConfig = new StatsdConfig() { StatsdServerName = "127.0.0.1", StatsdPort = 8275 };
+            using (var dogStatsDService = new DataDogStatsdEventLogger(statsdConfig))
```

Comment:
is this going to be disposed before it can be used

Reply:
unfortunately, I think it is. I had better change this.